### PR TITLE
Merge dev to main

### PR DESCRIPTION
**Pull request by colearendt**

### CODE REVIEW

File: .github/actions/build-test-scan-push/action.yaml [hunk 1]
```diff
@@ -56,11 +56,16 @@ runs:
     - name: Set up Docker Buildx
       uses: docker/setup-buildx-action@v2
 
-    - name: Free Disk space
-      shell: bash
-      run: |
-        sudo rm -rf /usr/local/lib/android  # will release about 10 GB
-        sudo rm -rf /usr/share/dotnet # will release about 20GB
+    - name: Free Disk Space (Ubuntu)
```
Comment (aronatkins) [inline .github/actions/build-test-scan-push/action.yaml:59]: Some of the Connect workflows performed the same file removal commands that happened previously. We saw those steps take up to 10 minutes just for file purging!! This was unexpected. It's possible that it was a temporary issue with the runners.

Not suggesting any change, but sharing in case CI-time starts to become an issue.

File: NEWS.md [hunk 2]
```diff
@@ -5,6 +5,10 @@ changed in each image.
 
 This file only captures pervasive, repository-wide changes.
 
+# 2024-03-01
+
+- BREAKING: Update Quarto version to 1.4.550
```
Comment (aronatkins) [inline NEWS.md:10]: This update needs to be coordinated with a change to the Connect Helm chart, as it currently references `/opt/quarto/1.3.340`.

File: connect-content-init/Dockerfile.ubuntu2204 [hunk 3]
```diff
@@ -10,7 +10,7 @@ RUN apt-get update && \
     apt-get install -y --no-install-recommends ca-certificates curl && \
     rm -rf /var/lib/apt/lists/*
 
-ARG RSC_VERSION=2024.01.0
+ARG RSC_VERSION=2024.02.0
```
Comment (aronatkins) [inline connect-content-init/Dockerfile.ubuntu2204:13]: Connect is already using 2024.02.0 on main; why is this change here? Does `dev` need to receive `main` ahead of this work?

File: NEWS.md (hunk 2 above)
Comment (aronatkins) [reply NEWS.md:10]: CC @kgartland-rstudio

File: connect-content-init/Dockerfile.ubuntu2204 (hunk 3 above)
Comment (bschwedler) [reply connect-content-init/Dockerfile.ubuntu2204:13]: My guess is this is showing up due to the commit being different in each branch, even though the commits have the same content.

I created #710 to get `dev` up-to-date with `main`, which should hopefully resolve this issue.

Comment (bschwedler) [reply connect-content-init/Dockerfile.ubuntu2204]: This looks to be resolved.

File: NEWS.md (hunk 2 above)
Comment (bschwedler) [reply NEWS.md:14]: This will be resolved through #711

**Review by bschwedler — APPROVED** [review]
LGTM

We are discussing the best way to handle the daily workbench installation failures.
